addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/SpiderPanel.java

```diff
@@ -444,7 +444,7 @@ public void spiderStarted() {}
     public void foundMessage(
             HistoryReference historyReference, HttpMessage httpMessage, ResourceState state) {
         boolean added = addHistoryUrl(historyReference, httpMessage, targetSite, state);
-        if (View.isInitialised() && added) {
+        if (View.isInitialised() && state == ResourceState.PROCESSED && added) {
```

Comment:
Does this still behave if there is no scope?

Reply:
Yes